### ISSUE TITLE
Gerald Cutting Sloppy - Play in the cutter head

### ISSUE THREAD

**Issue by LeeCarmichael** [DoESLiverpool/somebody-should#1467]
I've come in this morning and Gerald is cutting sloppy.
On inspection, the laser cutter head has lots of play in it where it attaches to the X-Axis.
I had a look and couldn't see how this is tightened so will leave it so Sean/Mike etc.

You could get away with it for cutting profiles at slow speeds but my designs require engraving at approx 200 speed and the result was very rough.

I've attached an image to show the result and a video of the play.
Normally, this engraving would be very crisp.

![Screenshot 2020-07-12 at 09 04 30](https://user-images.githubusercontent.com/45716447/87241826-b8fef100-c41e-11ea-9407-1b0c55e0f7d4.png)

Video - https://drive.google.com/file/d/10oZMC3f_OSxds4ORs94OzGVKgran3Jf6/view?usp=sharing

If this gets resolved, can someone please text me on 07495943465 and let me know. I'm desperate to make orders on Gerald right now.

Thanks.

**Comment by jltomo** [comment]
Following - also needing to cut to fulfil orders

**Comment by johnmckerrell** [comment]
Are you able to use Sophia? I believe both are available you just can’t book Sophia because only one household can be in the room at  once. 

--

**Comment by jltomo** [comment]
The raw sheets I have are too big for Sophia, if she’s operating I could but I’d have to cut them down...!

**Comment by LeeCarmichael** [comment]
My sheets are also too big for Sophia. Every time I’ve tried to use Sophia something has gone wrong too.

**Comment by amcewen** [comment]
Is it any better if the bi-directional engraving is turned off?  That can work round backlash in the mechanism because you're only engraving in one direction so it's always at the same "end" of the backlash.  The only downside is that the engraving takes twice as long.

It's hard to tell from the photo, but it (also) looks like the scan gap is a bit off.  Has someone maybe changed that?  The laser is 0.2mm wide, so usually you'd want the engraving scan gap to be 0.2 or 0.1mm (IIRC)

**Comment by LeeCarmichael** [comment]
Didn’t try the one direction engraving as it doubles the cutting time. The scan gap was actually 0.05 which is what I always use, the slop in the cutter head has created what looks like a large scan gap.

**Comment by magman2112** [comment]
I also noticed that this main bearing was poorly when I was last in DoES, but it is not a small repair. 

I have already bought a new rail and bearings, but it is about 200mm too long to fit Gerald. As this is a hardened steel rail, it needs an angle grinder and metal cutting disc to cut it, which I don’t currently own. I do plan to pick one up this week though, to see if we can move forward with this repair. 

We can also possibly try just replacing the bearing, using one of the new ones, but there is no guarantee that the new bearing is compatible with the rail currently installed in Gerald. This would likely be a quicker interim repair though.

We do plan a possible enhancement to Gerald as well, using 2 bearings on this axis, for additional rigidity, but this will take a little bit longer again than the main repair, as a bit more engineering is required, but it is on our radar.

It looks like Gerald is booked on Monday all day, so I will see if I can get in on Tuesday morning to have a look at this.

Also, if you include @DoESLiverpool/laser-maintenance in your GitHub post, then a notification is sent out to all of the Laser Maintenance team, which might speed up any repairs.

**Comment by KarlDunkerley** [comment]
Can't help with the engineering but give me a shout if there is anything else I can do.
Visors finished now so more spare time.

Sent from Yahoo Mail on Android 
 
  On Sun, 12 Jul 2020 at 15:25, Mike Gorman<notifications@github.com> wrote:   


I also noticed that this main bearing was poorly when I was last in DoES, but it is not a small repair.

I have already bought a new rail and bearings, but it is about 200mm too long to fit Gerald. As this is a hardened steel rail, it needs an angle grinder and metal cutting disc to cut it, which I don’t currently own. I do plan to pick one up this week though, to see if we can move forward with this repair.

We can also possibly try just replacing the bearing, using one of the new ones, but there is no guarantee that the new bearing is compatible with the rail currently installed in Gerald. This would likely be a quicker interim repair though.

We do plan a possible enhancement to Gerald as well, using 2 bearings on this axis, for additional rigidity, but this will take a little bit longer again than the main repair, as a bit more engineering is required, but it is on our radar.

It looks like Gerald is booked on Monday all day, so I will see if I can get in on Tuesday morning to have a look at this.

Also, if you include @DoESLiverpool/laser-maintenance in your GitHub post, then a notification is sent out to all of the Laser Maintenance team, which might speed up any repairs.

**Comment by JackiePease** [comment]
![IMG_20200712_162917](https://user-images.githubusercontent.com/12802256/87250501-38abb080-c45d-11ea-8786-5d7cae84ffd1.jpg)

I'm removing the broken label as I think Gerald is still useable for less demanding tasks. But I'll get in touch with whoever is booked in tomorrow to tell them the situation. 

First token cut with speed 250 - very fuzzy as reported by @LeeCarmichael 
Second with speed reduced to 150, third with speed reduced to 150 and unidirectional - still a little bit fuzzy but not too bad.

**Comment by JackiePease** [comment]
![IMG_20200712_162917](https://user-images.githubusercontent.com/12802256/87251038-cccb4700-c460-11ea-89a6-80bf092ffb33.jpg)

**Comment by JackiePease** [comment]
I've removed the Broken label as I think Gerald is still useable for less demanding work. I've DMd Jade to say that he isn't working well but she still intends to come in and try.
1st token in photo above had speed of 220, 2nd had speed of 150, 3rd had speed of 150 and unidirectional engraving. It's still a bit fuzzy but much better than the higher speeds.

**Comment by Sean-anotherone** [comment]
I have a grinder with metal cutting discs, but I thought those new bearings were for the y axis (there are 2?)
I could come over tomorrow after work?  (i.e. Monday)
Sean
    On Sunday, 12 July 2020, 15:25:39 BST, Mike Gorman <notifications@github.com> wrote:  
 
 


I also noticed that this main bearing was poorly when I was last in DoES, but it is not a small repair.

I have already bought a new rail and bearings, but it is about 200mm too long to fit Gerald. As this is a hardened steel rail, it needs an angle grinder and metal cutting disc to cut it, which I don’t currently own. I do plan to pick one up this week though, to see if we can move forward with this repair.

We can also possibly try just replacing the bearing, using one of the new ones, but there is no guarantee that the new bearing is compatible with the rail currently installed in Gerald. This would likely be a quicker interim repair though.

We do plan a possible enhancement to Gerald as well, using 2 bearings on this axis, for additional rigidity, but this will take a little bit longer again than the main repair, as a bit more engineering is required, but it is on our radar.

It looks like Gerald is booked on Monday all day, so I will see if I can get in on Tuesday morning to have a look at this.

Also, if you include @DoESLiverpool/laser-maintenance in your GitHub post, then a notification is sent out to all of the Laser Maintenance team, which might speed up any repairs.

**Comment by Sean-anotherone** [comment]
Plus, while Mike & have not actually ranked the repairs list yet,  I guess this just jumped to the top.
Sean


    On Sunday, 12 July 2020, 15:25:39 BST, Mike Gorman <notifications@github.com> wrote:  
 
 


I also noticed that this main bearing was poorly when I was last in DoES, but it is not a small repair.

I have already bought a new rail and bearings, but it is about 200mm too long to fit Gerald. As this is a hardened steel rail, it needs an angle grinder and metal cutting disc to cut it, which I don’t currently own. I do plan to pick one up this week though, to see if we can move forward with this repair.

We can also possibly try just replacing the bearing, using one of the new ones, but there is no guarantee that the new bearing is compatible with the rail currently installed in Gerald. This would likely be a quicker interim repair though.

We do plan a possible enhancement to Gerald as well, using 2 bearings on this axis, for additional rigidity, but this will take a little bit longer again than the main repair, as a bit more engineering is required, but it is on our radar.

It looks like Gerald is booked on Monday all day, so I will see if I can get in on Tuesday morning to have a look at this.

Also, if you include @DoESLiverpool/laser-maintenance in your GitHub post, then a notification is sent out to all of the Laser Maintenance team, which might speed up any repairs.

**Comment by magman2112** [comment]
The reason there are two rails is that it was considerably cheaper to buy 2 rails and 4 bearings from a European supplier than to buy a single rail and bearing from a UK supplier. The only snag is that they were too long.

More toys for us to play with after the repair perhaps?

I will see you Monday evening then. I will bring in my 1/2” socket set, as I expect some of the Allen bolts in the current rail may need a bit more leverage than an Allen key alone can provide.

**Comment by magman2112** [comment]
Sean and I replaced the horizontal rail and bearing on Gerald this evening as well as completing a full alignment. Gerald is now cutting well again.

The new rail was 4mm thicker than the old rail though, so the original screws that fixed the rail to the carriage were too short to attach the new rail to the carriage. We found some other screws in the space that were good enough for now - to get the new rail useable, but we will need to order new screws to completely attach the new rail, so I will leave this issue open for now till the new screws arrive and are fitted.

Sean also found that the laser test button on the control panel was set for too sort a duration, so increased the duration. This should help with laser test fires in the future, so only a single press is required now, rather than the multiple presses needed before.
